packages/base/any/kernels/5.10-lts/patches/series.arm64

```diff
@@ -16,3 +16,4 @@
 0020-accton-as5114.patch
 0021-tn48m-add-sfp-eeprom-support-to-ethtool.patch
 #0022-marvell-wa-tx-disable.patch
```

Comment:
This Line:
#0022-marvell-wa-tx-disable.patch
should be removed and you can use:   
**0022**-delta-tn48m-dn-series-dts.patch

Reply:
@Mickey201, No problem, updated in https://github.com/dentproject/dentOS/pull/45/commits/e63345162b252ee5bc6000a2fd4a7e9e5cce2453. Thanks for the suggestions.